api/v1/flow/flow.proto

```diff
@@ -350,6 +352,18 @@ enum IPVersion {
     IPv6 = 2;
 }
 
+message Tunnel {
+    enum Protocol {
+        NONE = 0;
```

Comment:
Just checking -- there are no cases when `UNKNOWN` is useful right? As in `NONE === UNKNOWN` in all cases?

Reply:
Yes. However, I took another look and semantically I'd prefer `Unknown`, since it refers to the `ProtocolType` and not to the whole `Tunnel` message.
Given I need to rebase to pick up CI changes, I'm going to rename that value too.
Thanks!